.gitlab/llnl-ci-conda.yml

```diff
@@ -142,23 +79,58 @@ conda-tests:
         python3 ci/tests/test.py -n $TEST -c "bash ci/tests/${TEST}/test.sh";
       done
     - python3 ci/tests/test.py -e
-
-.conda-env-cleanup: &conda-env-cleanup
-  stage: cleanup
-  tags:
-    - ruby
-    - shell
-  script:
     - *finalize
 
-conda-cleanup-on-build-failure:
-  needs:
-    - job: conda-build
-  when: on_failure
-  <<: *conda-env-cleanup
-
-conda-cleanup-on-test-failure:
-  needs:
-    - job: conda-tests
-  when: on_failure
-  <<: *conda-env-cleanup
+
+
+
+conda_setup_quartz:
+  variables:
+    RUNNER_TYPE: "shell"
+  extends: [.on_quartz, .job_tags, .conda_setup]
+
+conda_env_setup_quartz:
+  variables:
+    RUNNER_TYPE: "shell"
+  needs: [ "conda_setup_quartz" ]
+  extends: [.on_quartz, .job_tags, .conda_env_setup]
+
+conda_build_quartz:
+  variables:
+    RUNNER_TYPE: "shell"
+  needs: [ "conda_env_setup_quartz" ]
+  extends: [.on_quartz, .job_tags, .conda_build]
+
+conda_tests_quartz:
+  variables:
+    RUNNER_TYPE: "batch"
+    LLNL_SLURM_SCHEDULER_PARAMETERS: "--nodes=1 -t 60"
+  needs: [ "conda_build_quartz" ]
+  extends: [.on_quartz, .job_tags, .conda_tests]
+
+
+
+
+conda_setup_ruby:
+  variables:
+    RUNNER_TYPE: "shell"
+  extends: [.on_ruby, .job_tags, .conda_setup]
```

Comment:
this one is not needed, since quartz will install conda (machines with the same arch, Lassen will have `conda_setup_lassen`)